From 8c8154bc9cdfb4cba19f1f3a3eba05b602a70ccd Mon Sep 17 00:00:00 2001
From: barryclark <barry@barryclark.co>
Date: Sun, 6 Mar 2016 22:58:14 -0500
Subject: [PATCH 103/214] Move to Kramdown and Rouge

Jekyll 3 and GitHub Pages now only support Kramdown as the Markdown
engine, and Rouge as the syntax highlighter.

Ref:
https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-je
kyll-3-0

This commit updates Jekyll Now to support these changes, with just a
small _config.yml update.
---
 _config.yml | 12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/_config.yml b/_config.yml
index f62e5d0..dad6b74 100644
--- a/_config.yml
+++ b/_config.yml
@@ -53,13 +53,21 @@ baseurl: ""
 # !! You don't need to change any of the configuration flags below !!
 #
 
-markdown: redcarpet
-highlighter: pygments
 permalink: /:title/
 
 # The release of Jekyll Now that you're using
 version: v1.1.0
 
+# Jekyll 3 now only supports Kramdown for Markdown
+kramdown:
+  # Use GitHub flavored markdown, including triple backtick fenced code blocks
+  input: GFM
+  # Jekyll 3 and GitHub Pages now only support rouge for syntax highlighting
+  syntax_highlighter: rouge
+  syntax_highlighter_opts:
+    # Use existing pygments syntax highlighting css
+    css_class: 'highlight'
+
 # Set the Sass partials directory, as we're using @imports
 sass:
   style: :expanded # You might prefer to minify using :compressed
-- 
2.17.1

